src/utils/rest-api.js

```diff
@@ -445,6 +445,82 @@ export function fetchLoadInfos(studyUuid, selectedNodeUuid, loadId) {
     );
 }
 
+export function fetchLineInfos(studyUuid, selectedNodeUuid, lineId) {
```

Comment:
Why not calling directly fetchEquipmentInfos in all the dialogs ?

Reply:
removed